packages/appcd-request/README.md

```diff
@@ -1,33 +1,44 @@
 # appcd-request
 
-Appc Daemon service for making HTTP requests.
+Wrapper around [request](https://npmjs.org/package/request) that applies Appc Daemon configuration
```

Comment:
Should we document the environment variables and their usage at some point?

Reply:
Documented!